src/announcements/announcements.11tyData.js

```diff
@@ -0,0 +1,12 @@
+module.exports = {
+	author: "The A11Y Project Team",
+	category: "Announcement",
+	layout: "post",
+	tags: ["announcements"],
+	eleventyComputed: {
+		eleventyNavigation: {
+			key: ({ category }) => category,
+			title: ({ title }) => title,
+		},
+	},
+};
```

Comment:
2️⃣ 🔍 

Same idea as 1️⃣ , except it hard-codes more values. Now, each announcement should only need a `title` and a `description` in its front matter.